radio/internal/_radio.scss

```diff
@@ -89,17 +89,13 @@ $_md-sys-motion: tokens.md-sys-motion-values();
       );
     }
 
-    // <input> is also the touch target
-    input {
-      appearance: none;
+    .touch-target {
       height: 48px;
-      margin: 0;
       position: absolute;
       width: 48px;
-      cursor: inherit;
     }
 
-    :host([touch-target='none']) input {
+    :host([touch-target='none']) .touch-target {
       width: 100%;
       height: 100%;
```

Comment:
Things look good! The one thing I see to improve is we can `display: none` the touch target instead of making it fit the width/height, like we do with [button's touch target](https://github.com/material-components/material-web/blob/22e19c264a56241d84dabeb399dbfc2ebc352a63/button/internal/_touch-target.scss#L21). Mind making that change?

```css
:host([touch-target='none']) .touch-target {
  display: none;
}
```

Reply:
Understood. I will make the changes and get back to you shortly.